packages/engine-core/src/tasks/task.rnv.new.ts

```diff
@@ -665,6 +665,10 @@ export const taskRnvNew = async (c: RnvContext) => {
         ...renativeTemplateConfigExt,
         projectName: data.projectName || 'my-project',
         projectVersion: data.inputVersion || '0.1.0',
+        common: {
```

Comment:
@mihaiblaga89 id and title should be present in appConfig renative.json not project level. 
give the proper fix requires "bootstrap metadata" feature I pass it as temp workaround